compiler/integration-tests/test/integration/browser/compile_prove_verify.test.ts

```diff
@@ -4,14 +4,19 @@ import newCompiler, {
   compile,
   init_log_level as compilerLogLevel,
 } from "@noir-lang/noir_wasm";
+import { acvm, abi } from "@noir-lang/noir_js";
 import { Barretenberg, RawBuffer, Crs } from "@aztec/bb.js";
-import { acvm, noirc } from "@noir-lang/noir_js";
 import { decompressSync as gunzip } from "fflate";
-
+import { ethers } from "ethers";
 import * as TOML from "smol-toml";
 
+const mnemonic = "test test test test test test test test test test test junk";
+const provider = new ethers.JsonRpcProvider("http://localhost:8545");
+const walletMnemonic = ethers.Wallet.fromPhrase(mnemonic);
+const wallet = walletMnemonic.connect(provider);
```

Comment:
We should be verifying the proof in a `eth_call` context which removes the need for a full wallet. Not a blocker.